connect_database.py

```diff
@@ -8,19 +8,23 @@ def connect_mongo():
     db = mongo_client[str(DB_NAME)]
     return db
 
+def save_number(phone_number):
+    db = connect_mongo()
+    numbers = db["numbers"]
+    numbers.insert({"number":phone_number})
 
 def get_lifemap(phone_number):
     db = connect_mongo()
     familyCol = db['family']
     family = familyCol.find_one({'phoneNumber': phone_number}) # .Family needs to be the name of the collection where the familys are loaded
     if family :
         print(family)
-        if 'lifeMap' in family: 
+        if 'lifeMap' in family:
             print(family['lifeMap'])
```

Comment:
Is this `lifeMap` not `lifemap`? Just to double check

Reply:
changed to lifemapPdf